### ISSUE TITLE
Add support for HTTP web requests that return an array of uint8

### ISSUE THREAD

**Issue by prestonvasquez** [alpstable/gidari#189]
Some HTTP requests will return an array of uint8, representing images, etc. Here is the example we should use in our internal/transport package integration tests:

```sh
curl --location --request GET 'https://logo.clearbit.com/postman.com'
```
The config file would look like this:

```yml
url: https://logo.clearbit.com
connectionStrings:
  - "mongodb://mongo1:27017/logos"
  - "postgresql://root:root@postgres1:5432/defaultdb?sslmode=disable"
rateLimit:
  burst: 1
  period: 1
requests:
  - endpoint: /postman.com
    table: postman_imgs
```

This will require a new type to be added to to the [tools.Records routines](https://github.com/alpine-hodler/gidari/blob/main/tools/records.go#L269) to decode the byte data into a slice of structs for inserting into storage. The structs should probably have this shape:

```
{ Data: byteSlice}
```

This will require that we add a new [init table for postgres](https://github.com/alpine-hodler/gidari/blob/main/scripts/pg_create_tables.sql) to hold bytes. [Here](https://www.postgresql.org/docs/7.4/jdbc-binary-data.html#:~:text=PostgreSQL%20provides%20two%20distinct%20ways,type%20oid%20in%20your%20table.) are the PG docs.

**Comment by rish-singhal** [comment]
Hi! I can work on this issue, could you assign me please. Thanks!

**Comment by prestonvasquez** [comment]
@rish-singhal it's yours!

**Comment by prestonvasquez** [comment]
@rish-singhal Are you still interested in pursuing this issue?

**Comment by rish-singhal** [comment]
> @rish-singhal Are you still interested in pursuing this issue?

Yes!

**Comment by prestonvasquez** [comment]
@rish-singhal Going to unassign you until #191 is completed, in case someone else wants to pick this up.

**Comment by rish-singhal** [comment]
@prestonvasquez I have created the PR for #191, can I get this assigned to me now? Thanks!

**Comment by prestonvasquez** [comment]
Closed by #360
